### PR TITLE
Removed pullsling2 and changed pullsling1 to pullsling

### PR DESCRIPTION
**Pull request by Albertlw**

### CODE REVIEW

File: src/RobotMap.h [hunk 1]
```diff
@@ -28,7 +28,7 @@ class RobotMap {
 	  static const int kIDLeftArm = 50;
 	  static const int kIDRightArm = 51;
 	//.........Sling.........
-	  static const int kIDPullSling1 = 7;
+	  static const int kIDPullSling = 7;
 	  static const int kIDPullSling2 = 8;
```
Comment (duane701) [inline src/RobotMap.h]: Remove `kIDPullSling2`

File: src/Subsystems/Puncher.cpp [hunk 2]
```diff
@@ -20,7 +20,7 @@ std::shared_ptr<Puncher> Puncher::getInstance() {
 }
 
 Puncher::Puncher() : Subsystem(kSubsystemName),
-    pullSling1(RobotMap::kIDPullSling1),
+    pullSling(RobotMap::kIDPullSling),
     pullSling2(RobotMap::kIDPullSling2),
```
Comment (duane701) [inline src/Subsystems/Puncher.cpp]: Remove `pullSling2`

File: src/Subsystems/Puncher.cpp [hunk 3]
```diff
@@ -35,13 +35,13 @@ void Puncher::InitDefaultCommand() {
 }
 void Puncher::SetSlingShot(double mSpeed)
 {
-  pullSling1.Set(mSpeed);
+  pullSling.Set(mSpeed);
   pullSling2.Set(mSpeed);
```
Comment (duane701) [inline src/Subsystems/Puncher.cpp]: Remove `pullSling2`

File: src/Subsystems/Puncher.h [hunk 4]
```diff
@@ -17,7 +17,7 @@ class Puncher: public frc::Subsystem {
   static const char kSubsystemName[];
   static std::shared_ptr<Puncher> self;
 
-  WPI_TalonSRX pullSling1;
+  WPI_TalonSRX pullSling;
   WPI_TalonSRX pullSling2;
```
Comment (duane701) [inline src/Subsystems/Puncher.h]: Remove `pullSling2`

**Comment by Albertlw** [comment]
PTAL

**Review by duane701 — CHANGES_REQUESTED** [review]
FTP

**Review by duane701 — APPROVED** [review]
LGTM

**Pull request merged**